docs/post-call-survey.md

```diff
@@ -2,6 +2,67 @@
 
 You can configure post-call surveys to collect valuable feedback like quality of services or net promoter score after a call ends.
 
+## Create a 1-question poll
+
+### Database configuration
+
+One question poll survey results will be stored in a cosmosDB.
+Cosmos DB can be provisioned either by using DB connection string or endpoint. If both values are present, endpoint takes precedence.
+The data stored in DB is as follows:
+
+```
+{
+    callId: string,
+    acsUserId: string,
+    response: boolean | string | number //the response of the survey
+}
+```
+
+Logs related to the call can be found in Azure Monitor logs
+[Learn how to enable logging via Azure Monitor logs](https://learn.microsoft.com/en-us/azure/communication-services/concepts/analytics/enable-logging)
+
+CallId in the DB corresponsed to correlationId in Azure monitor logs
+
+### Configure using environment variables
+
+```
+VV_POSTCALL_SURVEY_TYPE = "onequestionpoll"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE = "Tell us how we did"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT = "How satisfied are you with this virtual appointment's audio and video quality?"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE = "likeOrDislike"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_ANSWER_PLACEHOLDER = ""
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT = "Continue"
+VV_COSMOS_DB_CONNECTION_STRING = "<your_cosmosDB_connectionString>"
+VV_COSMOSDB_ENDPOINT = "<your_cosmosDB_endpoint>"
+VV_COSMOSDB_NAME = "<your_cosmbosDB_name>"
+```
+
+### Configure using defaultConfig.json
+
+```
+{
+    ...
+    "postCall": {
+        "survey": {
+            "type": "onequestionpoll",
+            "options": {
+                "title": "Tell us how we did",
+                "prompt": "How satisfied are you with this virtual appointment's audio and video quality?",
+                "pollType": "likeDislike",
+                "answerPlaceholder": "",
+                "saveButtonText": "Continue"
+            }
+        }
+    },
+    "cosmosDB": {
+        "endpoint": "<Set the endpoint to authenticate using Managed Identity method>",
+        "connectionString": "<Or, set the connection string. If both values are defined, endpoint takes precedence.>",
+        "dbName": "<your_cosmosDB_name>"
+    }
+}
+
```

Comment:
Let's add a line saying:

`[Learn how to control access to CosmosDB using Managed Identities](https://learn.microsoft.com/en-us/azure/cosmos-db/managed-identity-based-authentication)`